input/fsh/aliases.fsh

```diff
@@ -55,3 +55,9 @@ Alias: $nz-purpose-of-use = https://nzhts.digital.health.nz/fhir/ValueSet/nz-pur
 Alias: $communication-status = http://hl7.org/fhir/event-status
 Alias: $communication-category = http://hl7.org/fhir/communication-category
 Alias: $communication-medium = http://terminology.hl7.org/CodeSystem/v3-ParticipationMode
+
+Alias: $hnz-communication-delivery-status-codes = https://fhir-ig.digital.health.nz/shared-care/CodeSystem/hnz-communication-delivery-status-codes
+Alias: $hnz-communication-delivery-status-valueset = https://fhir-ig.digital.health.nz/shared-care/ValueSet/hnz-communication-delivery-status-valueset
+//Purpose Of Use
+
+Alias: $hnz-participation-mode = https://fhir-ig.digital.health.nz/shared-care/CodeSystem/hnz-participation-mode-codes
```

Comment:
We should probably standardise the naming convention of code systems and aliases; I think using the name looks good, eg "Alias: $hnz-participation-mode-codes" here